t/smoke/conftest.py

```diff
@@ -54,6 +54,7 @@ def default_worker_tasks(default_worker_tasks: set) -> set:
     network="{default_pytest_celery_network.name}",
     wrapper_class=RedisContainer,
     timeout=REDIS_CONTAINER_TIMEOUT,
+    command=fxtr("default_redis_broker_command"),
```

Comment:
The `default_redis_broker_command` fixture configures Redis to boot with proper configurations.
Without it, the default config will collapse under the heavy use of the CI